dockerfiles/README.md

```diff
@@ -18,24 +62,22 @@
   docker run -it onnxruntime-ngraph
   ```
 
-## ONNX Runtime Server (Preview)
-#### Linux 16.04
+## TensorRT Version (Preview)
+#### Linux 16.04, TensorRT 5.1.2
```

Comment:
let's move to TensorRT 5.0.2
we will update to 5.1 once we can move ORT codebase to 5.1 branch of onnx-tensorrt

Reply:
Updated.